### ISSUE TITLE
Peer Review HW 2

### ISSUE THREAD

**Issue by glederrey** [agpmilli/adaepfl#1]
Hi guys,

I just give you some comments on my peer review.. It can maybe help you get better for the next ones.. =) 
1. In your **Male/Female Partitioning**, you had a graph for both female and male. It's really good. But don't forget to add the labels on the x and y axis.. I know we understand what they're talking about, but it's always better to have them.. ;-)
2. Still in the **Male/Female Partitioning**, you're missing the statistical tests. It's quite important. Because if we just check the average, we could come to the conclusion that these two values are significantly different. With the graph, we understand that statistically, it's not that significant but number always talk better than a graph.. ;-) 
3. In the exercise 2, you're also missing the statistical tests.. =)

In conclusion, even if you're missing the statistical tests, you did a good job.. I really like all the comments. It helps understanding what you did.. So, good job guys! =)
